config/locales/translation_ja.yml

```
@@ -9,3 +9,19 @@ ja:
         memo: メモ
         picture: 画像
         title: タイトル
+  views:
+    pagination:
+      first: "最初のページ"
+      last: "最後のページ"
+      previous: "前へ"
+      next: "次へ"
+      truncate: "..."
```

```suggestion
      truncate: "&hellip;"
```

実はドット3つと3点ピリオドは別の文字なんです。（調べてみてください）

---

ドットでいいやと思って打ち替えてしまいました💧 たまに検索していて目にしますが、これはエクリプスだったんですね。ちなみに下記のサイトで調べたのですが、「プログラミング英検」というのがあるのも初めて知りました。
https://styleguide.progeigo.org/docs/punctuation-symbol/ellipses/